.github/scripts/print_latest_commits.py

```diff
@@ -72,6 +72,9 @@ def get_commit_results(commit: str, results: Dict[str, Any]) -> List[Dict[str, A
 def isGreen(commit: str, results: Dict[str, Any]) -> Tuple[bool, str]:
     workflow_checks = get_commit_results(commit, results)
 
+    if len(workflow_checks) == 0:
```

Comment:
Could we restructure this code to make sure that there is at least one check run for each of the items in: 
     "pull+",
    "^trunk+",
    "^lint+",
    "^linux-binary+",
    "^android-tests+",
    "^windows-binary+"